app/assets/javascripts/util/js.cookies.js

```diff
@@ -0,0 +1,165 @@
+/*!
```

Comment:
Require the file in search.js.coffee and make it load right after jquery, which may require it to be moved to app/assets/javascripts

Reply:
Just moving the file to that directory loses it; where is it to be loaded right after jquery?

Reply:
There are three places that ```//= require jquery``` - ```search.js``` is the one that should require the cookie lib. Require the file after ```//= require jquery.color-2.1.2``` (line 4)

Reply:
Got it - I believe that'll work.